src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Control/AsyncType.fs

```diff
@@ -80,6 +80,15 @@ type AsyncType() =
         ()
 
     [<Test>]
+    member this.AsyncRunSynchronouslyReusesThreadPoolThread() =
+        let action = async { async { () } |> Async.RunSynchronously }
+        let computation =
+            [| for i in 1 .. 1000 -> action |]
+            |> Async.Parallel
+        Async.RunSynchronously(computation, timeout = 1000)
+        |> ignore
```

Comment:
Ok so the assertion is "no timeout" that's pretty nice, but maybe a small
comment for clarification would help.
On Dec 2, 2015 22:13, "radekm" <notifications@github.com> wrote:

> In
> src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Control/AsyncType.fs
> <https://github.com/Microsoft/visualfsharp/pull/678#discussion_r46476056>:
>
> > @@ -80,6 +80,15 @@ type AsyncType() =
> >          ()
> >
> >      [<Test>]
> > +    member this.AsyncRunSynchronouslyReusesThreadPoolThread() =
> > +        let action = async { async { () } |> Async.RunSynchronously }
> > +        let computation =
> > +            [| for i in 1 .. 1000 -> action |]
> > +            |> Async.Parallel
> > +        Async.RunSynchronously(computation, timeout = 1000)
> > +        |> ignore
>
> Yes, there is no assert. The purpose of this test is to ensure that the
> computation completes quickly (originally it took more than 1 minute with
> 100 actions - now there are 1000 actions and it takes less than 1 second) -
> otherwise TimeoutException would be raised.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Microsoft/visualfsharp/pull/678/files#r46476056>.
>